css/styles/style.css

```diff
@@ -0,0 +1,80 @@
+body {
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    max-width: 900px;
+    margin: 0 auto;
+    padding: 3rem;
+}
```

Comment:
i think its better if we remove max-width from body element and maybe padding also. you can achieve same task with wrapping your element with div tag and add all that styling there